.travis.yml

```diff
@@ -1,27 +1,20 @@
-language: node_js
+
+language: objective-c
+compiler: clang
 node_js: '0.10'
 env:
   global:
     - secure: kjz286d6rs2VHpv4WnxAOYXcdTIaTaABZ0SNXf27r5H+NDu5qx8ZSeCmj/9op/sDjTtpkLKhi2a0njUCWwak28RAWP3dNkT92MkUPVKBkevznSxWeqwLHoUos3AJnhPMB+cwrWvmRC+Ljt5K1q8SxQsS/HXM9pzKL3DG3qxM+eo=
-branches:
-  only:
-    - master
-notifications:
-  email:
-    - jonas@halfdanj.dk
+  matrix:
+    - NAME="OSX" BEFORE_THE_INSTALL="" SCRIPT_TO_RUN="scripts/osx/testTemplate.sh" AFTER_THE_SUCCESS="echo 'openFrameworks OSX Build Passing'"
```

Comment:
ok, but then wouldn't you better check for the correct name in the `after_success` section, and run the relevant code?